tests/tests/control_restart

```diff
@@ -58,6 +58,7 @@ export NSTF_NAME='2,0,0,0,0'
 export WRITE_DOPOST=.true.
 export IAER=5111
 export FHROT=12
+export FHRST=18
```

Comment:
The new variable FHRST is really just FHROT + SHOUR. I think it is clearer if you show where the 18 comes from. You could also just use that directly in the RESTART_FILE_PREFIX (or keep it $FHRST but set FHRST=$FHROT+$SHOUR 

```
export RESTART_FILE_PREFIX="${SYEAR}${SMONTH}${SDAY}.$(printf "%02d" $(( ${FHROT} + ${SHOUR} )))0000"
```

Reply:
I can change that.